andela/ah-jawans-frontend
# #166790068 Implements highlighting text in an article

#### What does this PR do?
Allow a user to be able to highlight and comment on any text in an article
#### Description of Task to be completed?
A user can highlight a text from an article
A user can comment on the highlighted text
#### How should this be manually tested?
Navigate to a particular article
Select a particular part and you will see a popup with a highlight button, then click that button
You will get a text box to add your comment and then send, and your text will be highlighted
#### What are the relevant pivotal tracker stories?
[#166790068](https://www.pivotaltracker.com/story/show/166790068)
#### Screenshots (if appropriate)
